src/writer.rs

```diff
@@ -802,7 +808,7 @@ mod tests {
 
     #[test]
     fn test_overflow_subtract() {
-        let overflow_size = std::u32::MAX / size_of::<FdtReserveEntry>() as u32 - 3;
+        let overflow_size = u32::MAX / size_of::<FdtReserveEntry>() as u32 - 3;
```

Comment:
I used the deprecated form here because the version of Rust in the CI Docker container was too old to support `u32::MAX`, but if it has been upgraded now, this looks good to me.

Reply:
It looks like it works with the CI, so I would say to merge it once we get another review.

Reply:
Sounds good, I probably just need to update my local copy of the docker container.

Reply:
The latest version is `v11`